### PR TITLE
#167482351 add tag row feature

### PR DESCRIPTION
**Pull request by frankly034**
#### What does this PR do?

- This PR creates TagRow component

#### Description of task to be completed?

- add tag row molecule component
- add tag row styleguide
- write test for tag row component

#### How should this be manually tested?

  - $ git clone https://github.com/andela/king-kong-ah-frontend.git
  - $ npm install
  - $ npm test

#### Any background context you want to provide?

- You would need npm to run this application

#### What are the relevant pivotal tracker stories?

- PT Story link - [#167482351](https://www.pivotaltracker.com/story/show/167482351)

#### Screenshots (if appropriate)

image

### CODE REVIEW

**Review by akinyeleolat — APPROVED** [review]

**Review by mosinmiloluwa01 — CHANGES_REQUESTED** [review]
good job @frankly034, but it'll be nice if you add a scrolling feature to the tag row since it will be added directly on the homepage.

File: src/components/UI/molecules/TagRow/TagRow.jsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
+/* eslint-disable no-use-before-define */
```
Comment (EmmanuelSage) [inline src/components/UI/molecules/TagRow/TagRow.jsx:1]: Good job lewis,

I guess you are disabling this rule so you could use the es5 function without defining before you use it in your component,
This means you are hoisting the function to the top,

I would like to know if there is a special reason, you are relying on `Hoisting` for the renderTags() method??

Comment (frankly034) [reply src/components/UI/molecules/TagRow/TagRow.jsx:1]: Code clarity as advised by TTL as explained during the call

**Comment by benfluleck** [comment]
I told you to make one of these have an active tag

**Review by mosinmiloluwa01 — APPROVED** [review]

**Review by benfluleck — APPROVED** [review]

**Pull request merged**